thrifty-kotlin-codegen/src/main/kotlin/com/microsoft/thrifty/kgen/KotlinCodeGenerator.kt

```diff
@@ -259,6 +262,10 @@ class KotlinCodeGenerator(
         this.emitBigEnums = true
     }
 
+    fun numValuesPerHugeEnum(numValuesPerHugeEnum: Int): KotlinCodeGenerator = apply {
```

Comment:
For consistency with the other options, I wonder if this should also be named with an `emit` prefix, like `emitHugeEnumsWithSize`.